src/terminal.rs

```diff
@@ -1,25 +1,20 @@
-
 use std;
 use termion;
-
 use termrect::{RawPaintable, HasSize};
 use style::Style;
 use styledtext::StyledText;
 use std::io::Write;
 
 pub struct Terminal<W: Write> {
     size: (u32, u32),
-    current_style: Style,
     w: W,
 }
 
 impl<W: Write> Terminal<W> {
     pub fn new(mut w: W, size: (u32, u32)) -> Terminal<W> {
-        let current_style = Style::default();
-        write!(w, "{}", current_style).unwrap();
```

Comment:
Same here. Please leave current_style alone for now.

Reply:
Done!